src/api/posts/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  extends: '@senecacdot/eslint-config-telescope',
+
+  overrides: [
+    {
+      files: ['./**/*.js'],
+      env: {
+        jest: true,
+        commonjs: true,
+        es2021: true,
+        node: true,
+      },
+      rules: {
+        'no-shadow': 'off',
+        '@typescript-eslint/no-shadow': 'off',
+      },
+    },
+  ],
```

Comment:
After extending the base config, you can declare other configurations specific to this package like normal, so it's not overriding.
```suggestion
      env: {
        jest: true,
        commonjs: true,
        es2021: true,
        node: true,
      },
      rules: {
        'no-shadow': 'off',
        '@typescript-eslint/no-shadow': 'off',
      },
```